source/renderer/app/routes-config.js

```diff
@@ -8,6 +8,7 @@ export const ROUTES = {
   PROFILE: {
     LANGUAGE_SELECTION: '/profile/language-selection',
     TERMS_OF_USE: '/profile/terms-of-use',
+    DATA_LAYER_MIGRATION_ACCEPTANCE: '/profile/data-layer-migration',
```

Comment:
@daniloprates for the route key I would like to keep the format as used in TERMS_OF_USE - so that we have `DATA_LAYER_MIGRATION`. Without the "ACCEPTANCE" part...